### DISCUSSION TITLE
Welcome to authorization-server-openiddict Discussions!

### DISCUSSION THREAD

**Discussion by robinvanderknaap** [robinvanderknaap/authorization-server-openiddict#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BPA8** [comment]
@robinvanderknaap  Thank you for providing sample implementation for OpenIDDict authorization server.  You did mention in your blog [post ](https://dev.to/robinvanderknaap/setting-up-an-authorization-server-with-openiddict-part-iii-client-credentials-flow-55lp) that we can use our own custom stores in place of entityframework. I was wondering if you could provide pointers on how to wire up custom stores that implement asp.net Identity. What all components should be implemented and any samples would be greatly appreciated. Thanks

**Reply by robinvanderknaap** [reply]
Hi,

The sample in the OpenIddict repo uses ASP.NET identity for authentication, https://github.com/openiddict/openiddict-core/tree/dev/samples.

More samples are available in a separate repo: https://github.com/openiddict/openiddict-samples

**Comment by shadowfoxish** [comment]
Hi, I'm a .NET full stack developer, been doing this now for over 10 years. Always have a lot of crazy problems to solve, and never enough time! OpenIddict and this blog series definitely helped me out and learn a good bit about OpenID Connect and how I might be able to use them to solve some challenges at work. 

I opened a PR for consideration, which adds an external login provider and sort of closes the gap for folks not interested in using Aspnet Identity for a membership stack that supports SSO. I hope it helps the next person.
